Description: Pass selected files to gcr-viewer from its desktop entry
 Add the desktop-entry field code so selected files are passed to
 gcr-viewer when opened from a file manager.
Forwarded: not-needed
Last-Update: 2026-10-19

--- a/ui/gcr-viewer.desktop.in.in
+++ b/ui/gcr-viewer.desktop.in.in
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=View file
 MimeType=application/pkcs12;application/pkcs12+pem;application/pkcs7-mime;application/pkcs7-mime+pem;application/pkcs8;application/pkcs8+pem;application/pkix-cert;application/pkix-cert+pem;application/pkix-crl;application/pkix-crl+pem;application/x-pem-file;application/x-pem-key;application/x-pkcs12;application/x-pkcs7-certificates;application/x-x509-ca-cert;application/x-x509-user-cert;application/pkcs10;application/pkcs10+pem;application/x-spkac;application/x-spkac+base64;
-Exec=@bindir@/gcr-viewer
+Exec=@bindir@/gcr-viewer %F
 Type=Application
 Terminal=false
 NoDisplay=true
